services/cinema_hall.py

```diff
@@ -0,0 +1,19 @@
+from db.models import CinemaHall
+import init_django_orm  # noqa: F401
+from django.db.models import QuerySet
+
+
+def get_cinema_halls() -> QuerySet:
+    return CinemaHall.objects.all()
+
+
+def create_cinema_hall(
+        hall_name: str,
+        hall_rows: int,
+        hall_seats_in_row: int
+) -> None:
```

Comment:
It's a good practice to return something from functions such as `create_...`, `update_...`, etc.

Reply:
Understand.